### PR TITLE
convert docs for Projects and Response fields

### LINKED ISSUE #2: Convert to https://github.com/tripit/slate

### PR DESCRIPTION
**Pull request by kariannemah**
Closes #2.

The old docs had an example response for each example request, but in `source/includes/responses.md`, example responses are shown for only "List a project's response" and "Get a single response." 

I followed your lead: `source/includes/projects.md` and `source/includes/project-forms.md` list example responses for GET requests only.

Just curious—why'd you decide to leave them out?

### CODE REVIEW

**Comment by kariannemah** [comment]
And a general question about using cURL to interact with an API:

Is it cURL syntax that requires you to append a param to the path when making a POST request? 
http://take.ms/eczME

**Comment by ajb** [comment]
> Just curious—why'd you decide to leave them out?

a) I couldn't figure out how to get partials working properly, and
b) it was redundant -- we should just say "it returns a `Response` resource" and then have a definition, ideally just one (not two like I did), of what that resource looks like.

> Is it cURL syntax that requires you to append a param to the path when making a POST request? 

I'm not sure I understand. Saying `GET /projects/:project_id/response/:response_id` is just a convention for using variables in a path, the equivalent of `GET /projects/$project_id/response/$response_id`.

We should probably replace the path in the curl examples with a full, working request, instead of the pseudo-request that's currently in there. (See how Stripe does it: http://take.ms/C81PV)

**Comment by kariannemah** [comment]
I understand that convention. Hmm, let me rephrase:

For "Create a response," the HTTP request is `POST /projects/:project_id/responses` but the curl command is `curl /projects/:project_id/responses/:response_id ....` as seen here: http://take.ms/Yfdv7

Why does the latter need `:response_id` tacked on to the end of the path?

I'm with you on full, working curl examples, btw— :+1:

**Comment by ajb** [comment]
Ah. http://take.ms/Yfdv7 is a typo, simple as that :D. The only requests which need a trailing `:id` parameter are `PUT` and `DELETE`.

**Comment by kariannemah** [comment]
(and when we want to retrieve a specific record with GET, right?)

ty for clarifying! :-)

**Comment by ajb** [comment]
err yes, whoops

On Tuesday, April 28, 2015, Kari Mah <notifications@github.com> wrote:

> (and when we want to retrieve a specific record with GET, right?)
>
> ty for clarifying! :-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dobtco/screendoor-api-docs/pull/8#issuecomment-97269193>
> .
>


-- 
Adam Becker
(951) 9-BECKER
@AdamJacobBecker
